src/cascadia/TerminalSettingsModel/ActionArgs.cpp

```diff
@@ -765,6 +766,14 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         return RS_(L"ResetWindowNameCommandKey");
     }
 
+    winrt::hstring SearchForTextArgs::GenerateName() const
+    {
+        return winrt::hstring{
+            fmt::format(std::wstring_view(RS_(L"SearchForTextCommandKey")),
+                        QueryUrl().c_str())

```

Comment:
note if we go with the %s idea we may want to revisit how this is phrased

Reply:
We could consider automatically generating a short version by parsing the URL and turning it into like.. `Search {Uri().Hostname()}` => `Search google.com"

(it is not unambiguous, but it could be interesting?)

Reply:
changed it, but should we be careful if Uri could throw if queryUrl is malformed?